hedera-node/hedera-app-spi/src/testFixtures/java/com/hedera/node/app/spi/fixtures/Assertions.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.spi.fixtures;
+
+import com.hedera.hapi.node.base.ResponseCodeEnum;
+import com.hedera.node.app.spi.workflows.PreCheckException;
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/** A convenience class for testing with Hedera specific assertions */
+public final class Assertions {
```

Comment:
I'm not convinced that this class deserves to exist. Because of this class, we can use this code in the AdminService preHandle tests:
```
        assertPreCheck(() -> subject.preHandle(context, specialFileStore), INVALID_FREEZE_TRANSACTION_BODY);
```
instead of this code:
```
        PreCheckException e = assertThrows(PreCheckException.class, () -> subject.preHandle(context, specialFileStore));
        assertSame(INVALID_FREEZE_TRANSACTION_BODY, e.responseCode());
```
so we're only saving one line of code per test and adding a bit to the cognitive overhead, since now you have to go into this class and see what it does to understand the test.

Reply:
Maybe the name of the function should be "assertThrowsPreCheck". The example you have uses JUnit assertions, but we've already all agreed to use AssertJ, which actually makes this a little longer and more cumbersome (requires reflection to check the status!). @netopyr has another PR where he did something even more clever by extending JUnit and I really liked his approach. although it is 3-4 lines per check instead of 1. Still, his approach might be better. I've asked him in his PR to figure out the best way to combine these approaches (or just remove this one is fine too)